Exploration Queries/InputEntity_Account/UserAccount_SubsOwnedDirectly.yaml

```diff
@@ -0,0 +1,24 @@
+Id: d103d814-eb0f-4395-af45-7f5697e598aa
+DisplayName: "Subscriptions owned by the user"
+Description: "Returns all subscriptions owned directly by the user"
+InputEntityType: Account 
+InputFields:
+  - AadUserId 
+OutputEntityTypes:
+  - AzureResource  
+QueryPeriodBefore: 7d 
+QueryPeriodAfter: 0
+DataSources:
+  - UserAccessAnalytics
+Tactics:
+  - LateralMovement 
+query: |  
+    let GetUserOwnedSubscriptions = (v_Account_AadUserId:string) { 
+    UserAccessAnalytics  
+        | where SourceEntityId =~ v_Account_AadUserId
+        | where TargetEntityType == "AzureSubscription"
+        | where AccessLevel == "Owner"
+        | project TimeGenerated, AzureResource_ResourceId = strcat("/subscriptions/", TargetEntityId), AzureResource_SubscriptionId = TargetEntityId, AzureResource_Aux_SubsName = TargetEntityName
+        | summarize arg_max(TimeGenerated, *) by AzureResource_ResourceId, AzureResource_SubscriptionId, AzureResource_Aux_SubsName

```

Comment:
Generally we want to do Top 5 or 10 here, is the expectation that we won't have more than that normally?